hedera-node/test-clients/src/main/resource/contract/contracts/OpcodesContract/OpcodesContract.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.8.18;
+
+// to build
+// solc OpcodesContract.sol --ir > OpcodesContract.yul
+// tweak the push 0 yul
+// delete the IR: line
+// solc --strict-assembly OpcodesContract.yul --bin > OpcodesContract.bin
+// solc --strict-assembly OpcodesContract.yul --abi > OpcodesContract.json
+
+contract NewOpcodes {
+
+
+    function opShl(uint _one, uint _two) public pure returns (uint _resp){
+        assembly {
+            _resp := shl(_one, _two)
+        }
+    }
+
+    function opShr(uint _one, uint _two) public pure returns (uint _resp){
+        assembly {
+            _resp := shr(_one, _two)
+        }
+    }
+
+    function opSar(uint _one, uint _two) public pure returns (uint _resp){
+        assembly {
+            _resp := sar(_one, _two)
+        }
+    }
+
+    function opExtCodeHash(address _addr) public view returns (bytes32 _resp){
+        assembly {
+            _resp := extcodehash(_addr)
+        }
+    }
+
+    function opPush0() public pure returns (uint _resp) {
+        assembly {
+        _resp := add(msize(), 0x5f)
+//      _resp := add(verbatim_0i_1o(hex"5f"), 0x5f)
```

Comment:
nit: remove 
nit2: new line at end of file :)

Reply:
No, this should not be removed.  It is an interim step because (a) solidity doesn't support the push0 opcode in assembly yet and (b) doesn't support the verbatim in solidity.  So this is a placeholder to show what should be added.

NewOpcodes.sol (from which this is derived) didn't have a newline at the end of a file.